### PR TITLE
Philips TV ambilight support

### PR DESCRIPTION
**Pull request by elupus**
## Proposed change

Update documentation for ambilight support.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/44867

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/philips_js.markdown [hunk 1]
```diff
@@ -111,3 +115,14 @@ The integration provides a remote entity for sending remote key presses directly
 | Online           |                                           |
 | SmartTV          |                                           |
 | PhilipsMenu      |                                           |
+
+### Ambilight
+
+The integration exposes a single light entity to control the mode of the ambilight on the TV. It allow setting a fixed background color or switching the TV to one of the lounge modes supported by the TV.
```
Comment (frenck) [inline source/_integrations/philips_js.markdown]:
```suggestion
The integration exposes a single light entity to control the mode of the ambilight on the TV. It allows setting a fixed background color or switching the TV to one of the lounge modes supported by the TV.
```

File: source/_integrations/philips_js.markdown [hunk 2]
```diff
@@ -111,3 +115,14 @@ The integration provides a remote entity for sending remote key presses directly
...
+
+When the light entity is turned on, it is controlling the ambilights, when it is turned off the TV is in control of the ambilight in it's standard video based fashion.
```
Comment (frenck) [inline source/_integrations/philips_js.markdown]:
```suggestion
When the light entity is turned on, it is controlling the ambilights, when it is turned off the TV is in control of the ambilight in its standard video-based fashion.
```

File: source/_integrations/philips_js.markdown [hunk 3]
```diff
@@ -111,3 +115,14 @@ The integration provides a remote entity for sending remote key presses directly
...
+
+Limits:
+ - The integration does not expose current ambilight measured values since it would
+overload the event bus in Home Assistant.
+ - There is no support to control the standard, non expert, styles of the TV
```
Comment (frenck) [inline source/_integrations/philips_js.markdown]:
```suggestion
 - There is no support to control the standard, non-expert, styles of the TV.
```

**Comment by frenck** [comment]
@elupus There is a conflict, I'm not sure how to resolve it correctly, could you take a look?

**Comment by elupus** [comment]
Fixed

**Comment by frenck** [comment]
Thanks, @elupus 👍

**Review by frenck — APPROVED** [review]

**Pull request merged**